packages/ckeditor5-dev-env/lib/release-tools/utils/getpackagestorelease.js

```diff
@@ -79,4 +83,17 @@ module.exports = function getPackagesToRelease( options ) {
 
 			return Promise.resolve( packagesToRelease );
 		} );
+
+	function displaySkippedPackages( skippedPackages ) {
+		if ( skippedPackages && !skippedPackages.length ) {
+			return;
+		}
+
+		const { logger } = require( '@ckeditor/ckeditor5-dev-utils' );
+
+		let message = 'Packages listed below have been skipped:\n';
+		skippedPackages.forEach( ( packageName ) => message += `  * ${ packageName }\n` );
```

Comment:
This would be better as a `map()` + `join()`. You wouldn't need to trim it afterwards.